dataactvalidator/scripts/load_sf133.py

```diff
@@ -61,6 +65,66 @@ def fill_blank_sf133_lines(data):
     return data
 
 
+def update_tas_id(fiscal_year, fiscal_period):
+    """Set the tas_id on newly SF133 entries. We use raw SQL as sqlalchemy
+    doesn't have operators like OVERLAPS and IS NOT DISTINCT FROM built in
+    (resulting in a harder to understand query)."""
+    sess = GlobalDB.db().session
+
+    # number of months since 0AD for this fiscal period
+    zero_based_period = fiscal_period - 1
+    absolute_fiscal_month = 12 * fiscal_year + zero_based_period - 3
+    absolute_following_month = absolute_fiscal_month + 1
+    start_date = date(absolute_fiscal_month // 12,
+                      (absolute_fiscal_month % 12) + 1,     # 1-based
+                      1)
+    end_date = date(absolute_following_month // 12,
+                    (absolute_following_month % 12) + 1,    # 1-based
+                    1)
+
+    # Why min()?
```

Comment:
So if there are multiple active tas entries, we just care about the one that was there first?